packages/contracts-bedrock/scripts/FeeVaultWithdrawal.s.sol

```diff
@@ -1,74 +1,76 @@
-// SPDX-License-Identifier: MIT
-pragma solidity 0.8.15;
+// TODO replace with RevenueSharer execution script
```

Comment:
Can we keep this script and just add a new one? If we remove it now, there are still people that depend on it